sbtc/tests/integration/validation.rs

```diff
@@ -386,3 +400,166 @@ fn op_csv_disabled() {
         err => panic!("{err}"),
     };
 }
+
+/// This validates that a user can reclaim their funds after the locktime.
+///
+/// The test proceeds as follows:
+/// 1. Spend all the user's funds for am sBTC deposit. Check that the
+///    balance is zero.
+/// 2. Wait locktime blocks after the first confirmation.
+/// 3. Create and submit another transaction reclaiming the funds.
+/// 4. Confirm the transaction and check that the balance is what it is
+///    supposed to be, less the bitcoin transaction fees.
+#[cfg_attr(not(feature = "integration-tests"), ignore)]
+#[test]
+fn reclaiming_rejected_deposits() {
```

Comment:
Should we also have a negative test? i.e. `cannot_reclaim_deposit_at_or_before_locktime()`? Or maybe this is covered somewhere else? Otherwise I guess it should be fairly easy to include it in this test prior to advancing past the lock time.

Reply:
Yeah let's test that case here, the one we have in `op_csv_disabled` is insufficient.

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/977/commits/33fb28ccd494a137025020c0ed4a2286a0ec215e